gateway/ln-gateway/src/db.rs

```diff
@@ -508,6 +523,82 @@ async fn migrate_to_v4(mut ctx: MigrationContext<'_>) -> Result<(), anyhow::Erro
     Ok(())
 }
 
+async fn migrate_to_v5(mut ctx: MigrationContext<'_>) -> Result<(), anyhow::Error> {
+    let mut dbtx = ctx.dbtx();
+    migrate_federation_config(&mut dbtx).await
+}
+
+async fn migrate_federation_config(
```

Comment:
Nit: Pluralize to `migrate_federation_configs()`

Reply:
Done